ethcore/light/src/cht.rs

```diff
@@ -26,10 +26,13 @@
 use ethcore::ids::BlockId;
 use ethereum_types::{H256, U256};
 use hashdb::HashDB;
+use keccak_hasher::KeccakHasher;
 use memorydb::MemoryDB;
 use bytes::Bytes;
-use trie::{self, TrieMut, TrieDBMut, Trie, TrieDB, Recorder};
+use trie::{TrieMut, TrieDBMut, Trie, TrieDB, Recorder};
+use ethtrie::{self, RlpCodec};
```

Comment:
(just commenting here because it's the first file I see this in)

It can be overly verbose to rely on 3 crates in order to use `patricia-trie-ethereum`. It would be easier to use if we re-exported the API of `keccak_hasher` and `patricia-trie` from `patricia-trie-ethereum`, maybe using `pub extern crate` declarations there.